### PR TITLE
file extension check for upload

### PR DESCRIPTION
**Pull request by leandrosveltestudios**

### CODE REVIEW

File: src/Gallery/Components/GalleryUploader.tsx [hunk 1]
```diff
@@ -1,27 +1,32 @@
...
+  const openFileInput = () => {
+    fileInputRef && fileInputRef.current && fileInputRef.current.click();
   };
 
   const selectFile: React.ChangeEventHandler = event => {
-    const imageUploaded: File = (event.target as HTMLInputElement)?.files[0];
-    upload(imageUploaded);
+    const fileUploaded: File = (event.target as HTMLInputElement)?.files[0];
+    const extension = fileUploaded.name.split('.')[1];
+    GalleryFileExtensions[type].includes(extension) ? upload(fileUploaded) : toast.promise(Promise.reject(), { error: 'Wrong file type' });
```
Comment (tjmkruger) [inline src/Gallery/Components/GalleryUploader.tsx:21]: Unsupported file type. Supported file types are ${}

File: src/Gallery/Components/GalleryFileExtensions.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export const GalleryFileExtensions = {
+  image: ['jpg', 'jpeg', 'png', 'webp'],
```
Comment (tjmkruger) [inline src/Gallery/Components/GalleryFileExtensions.ts:2]: I've never uploaded a Webp. It would be interesting to test to check that it works.

Comment (leandrosveltestudios) [reply src/Gallery/Components/GalleryFileExtensions.ts:2]: yes, `webp` works.

File: src/Gallery/Components/GalleryUploader.tsx (hunk 1 above)
Comment (leandrosveltestudios) [reply src/Gallery/Components/GalleryUploader.tsx]: Done

**Pull request merged**